common.py

```diff
@@ -0,0 +1,111 @@
+def get_month_year_from_string(string):
+    """
+    return month and year from combined string
+
+    month and year should be numbers and separated with /
+    Examples:
+        input string: 12/2006
+            output: 12, 2006
+        input string: jan/2006
+            output: None, None
+    """
+
+    month, year = string.split('/')
+    try:
+        month = int(month)
+        year = int(year)
+    except Exception:
+        print("Invalid input")
+        return None, None
+    return month, year
+
+
+def get_month_number_from_string(string):
+    """
+    Converts 3 characters month name to string
+    Example
+        input: "jan"
+            output: 1
+        input: "feb"
+            output: 2
+        input: "January":
+            output: None
+    """
+
+    string = str(string).lower()
+    if string == "jan":
+        return 1
+    elif string == "feb":
+        return 2
+    elif string == "mar":
+        return 3
+    elif string == "apr":
+        return 4
+    elif string == "may":
+        return 5
+    elif string == "jun":
+        return 6
+    elif string == "jul":
+        return 7
+    elif string == "aug":
+        return 8
+    elif string == "sep":
+        return 9
+    elif string == "oct":
+        return 10
+    elif string == "nov":
+        return 11
+    elif string == "dec":
+        return 12
```

Comment:
instead of if/else statements, you can have a `dict` containing month name as key and month number as month number. would make code more readable.

Reply:
Also, in case of a lot of `if/else` statements, please use `switch case` instead.